needy/platforms/android.py

```diff
@@ -145,3 +148,73 @@ def ndk_home(self):
 
     def default_architecture(self):
         return 'armv7'
+
+    def configuration(self, architecture):
+        api_level = 0
+        compiler = self.cxx_compiler(architecture)
+        binary_paths = self.binary_paths(architecture)
+        d = Platform.configuration(self, architecture)
+        d.update({
+            'api-level': self.__android_api_level(compiler, binary_paths),
+            'runtime': self.__runtime,
+            'runtime-version': {
+                'libc++-version': self.__libcpp_version(compiler, binary_paths),
+                'libc++-abi-version': self.__libcpp_abi_version(compiler, binary_paths)
+            } if self.__runtime == 'libstdc++' else {}
```

Comment:
and down here, the runtimes that use libc++ would be "c++_static" or "c++_shared". but in the case of standalone toolchains, you don't necessarily need to specify the runtime argument (so this should still work when `self.__runtime is None`

so i'd like to just make the case where `_LIBCPP_VERSION` is absent an expected case, with no warnings and good behavior